packages/ember-metal/lib/deprecate_property.js

```diff
@@ -22,7 +22,7 @@ import { set } from 'ember-metal/property_set';
 
 export function deprecateProperty(object, deprecatedKey, newKey) {
   function deprecate() {
-    Ember.deprecate(`Usage of \`${deprecatedKey}\` is deprecated, use \`${newKey}\` instead.`);
+    Ember.deprecate(`Usage of \`${deprecatedKey}\` is deprecated, use \`${newKey}\` instead.`, false, { id: 'ember-metal.deprecate-property', until: '3.0.0' });
```

Comment:
This is similar to deprecatingAlias. The calling sites need to be updated to pass through the deprecation options so id and until are unique.